### DISCUSSION TITLE
Difference in Patrick videos and Cyfrin Course.

### DISCUSSION THREAD

**Discussion by Harshvardhantomar** [Cyfrin/foundry-full-course-cu#1635] [General]
Hey, I've just started the course. I usually try to follow YouTube video, because  I feel that the website videos are somewhat laggy. But I saw that there is a foundry update in the course, which is not in the video. So, My question where can I track these updates to learn about them or should I start the course from the beginning to cover all the updates?

**Comment by EngrPips** [comment]
Hello @Harshvardhantomar, I will advise you to use Cyfrin's official website `https://updraft.cyfrin.io` to take all Cyfrin courses instead of YouTube as there would always be more on the Cyfrin official website and time-to-time updates the courses. I don't know where you can track the updates of courses on Cyfrin's official website other than the website itself. Courses on Cyfrin updraft are modular and are in chunks, so you don't have to start the course from the beginning but instead search through and find the content you need to watch to be updated on the course content

**Reply by Harshvardhantomar** [reply]
@EngrPips Thanks! Will do that.

**Reply by EngrPips** [reply]
You are welcome, friend.
